Mathlib/Algebra/Category/Ring/FilteredColimits.lean

```diff
@@ -187,21 +219,12 @@ def colimitCocone : Cocone F where
         RingHom.coe_inj ((Types.TypeMax.colimitCocone
           (F ⋙ forget CommSemiRingCat)).ι.naturality f) }
 
+
 /-- The proposed colimit cocone is a colimit in `CommSemiRingCat`. -/
-def colimitCoconeIsColimit : IsColimit <| colimitCocone.{v, u} F where
-  desc t := ofHom <|
-    (SemiRingCat.FilteredColimits.colimitCoconeIsColimit.{v, u}
-          (F ⋙ forget₂ CommSemiRingCat SemiRingCat.{max v u})).desc
-      ((forget₂ CommSemiRingCat SemiRingCat).mapCocone t) |>.hom
-  fac t j := hom_ext <|
-    RingHom.coe_inj <|
-      (Types.TypeMax.colimitCoconeIsColimit.{v, u} (F ⋙ forget CommSemiRingCat)).fac
-        ((forget CommSemiRingCat).mapCocone t) j
-  uniq t _ h := hom_ext <|
-    RingHom.coe_inj <|
-      (Types.TypeMax.colimitCoconeIsColimit (F ⋙ forget CommSemiRingCat)).uniq
-        ((forget CommSemiRingCat).mapCocone t) _ fun j => funext fun x =>
-        RingHom.congr_fun (congrArg Hom.hom (h j)) x
+def colimitCoconeIsColimit : IsColimit <| colimitCocone.{v, u} F := by
+  exact isColimitOfReflects (forget₂ _ SemiRingCat)
```

Comment:
Why is this `by exact` necessary? Can you put a short comment indicating the reason? Thanks.